VaultOfTheIncarnates/BroodkeeperDiurna.lua

```diff
@@ -169,6 +174,10 @@ function mod:OnEngage()
 	if self:GetOption(primalistMageMarker) then
 		self:RegisterTargetEvents("AddMarking")
 	end
+	
+	if self:GetOption(stormbringerMarker) then
+		self:RegisterTargetEvents("AddMarking2")
```

Comment:
Registering this if you also have primalistMageMarker enabled will overwrite the :RegisterTargetEvents, only 1 can be active. Thus you have to merge the 2 into 1 AddMarking function.